ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/execution/JsonRpcExecutor.java

```diff
@@ -102,7 +103,11 @@ public JsonRpcResponse execute(
 
   private Optional<JsonRpcError> validateMethodAvailability(final JsonRpcRequest request) {
     final String name = request.getMethod();
-    LOG.debug("JSON-RPC request -> {} {}", name, request.getParams());
+
+    if (LOG.isDebugEnabled()) {
+      final JsonArray params = JsonObject.mapFrom(request).getJsonArray("params");
+      LOG.debug("JSON-RPC request -> {} {}", name, params);
```

Comment:
too late, but next time can use `Slf4jLambdaHelper.debugLambda()`

Reply:
@garyschulte I did look at that but thought this read better since I wanted to guard the execution of the mapFrom function with isDebugEnabled...the same thing with the lambda was messier IMO